CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+Version 0.29.3
+-------------
+
+**Docs**
+- Add the changelog URL to the `pyproject.toml` file.
```

Comment:
```suggestion
- Add the changelog URL to the `pyproject.toml` project file so that the changelog reference is included under project links in PyPI.
```